test/vacations/vacation_test.exs

```diff
@@ -0,0 +1,33 @@
+defmodule GlassFactoryApi.Vacations.VacationTest do
```

Comment:
try to remember to write `@moduledoc false` if you are not writing documentation for a module =P

Reply:
TIL. Thanks <3<3<3